src/components/Note/index.tsx

```
@@ -0,0 +1,110 @@
+          aria-label={note.isFavorite ? "Desfavoritar" : "Favoritar"}
+        >
+          {note.isFavorite ? <ActiveFavoriteIcon /> : <FavoriteIcon />}
+        </button>
+      </div>
+
+      <Divider color={note.color !== "#FFF" ? "#FFF" : undefined} />
+
+      <div className={styles.noteBody}>
+        <div
+          style={{ width: "100%" }}
+          dangerouslySetInnerHTML={{ __html: note.body }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)